test/functional/mempool_accept.py

```diff
@@ -58,38 +55,30 @@ def check_mempool_result(self, result_expected, *args, **kwargs):
 
     def run_test(self):
         node = self.nodes[0]
+        miniwallet = MiniWallet(node)
 
         self.log.info('Start with empty mempool, and 200 blocks')
         self.mempool_size = 0
-        assert_equal(node.getblockcount(), 200)
+        miniwallet.generate(200)
+        assert_equal(node.getblockcount(), 400)
```

Comment:
the log 3 lines up says 200 blocks, but here we check 400. can you make these consistent?

Reply:
Got it! Fixed